ruleset/sca/amazon/cis_amazon_linux_2.yml

```diff
@@ -1347,9 +1373,9 @@ checks:
     condition: all
     rules:
       - 'c:sysctl net.ipv4.ip_forward -> r:^net.ipv4.ip_forward\s*=\s*0$'
-      - 'c:grep -RhEs "^\s*net\.ipv4\.ip_forward\s*=\s*0" /etc/sysctl.conf /etc/sysctl.d/*.conf -> r:net.ipv4.ip_forward\s*=\s*0'
+      - 'not c:grep -RhEs "net\.ipv4\.ip_forward" /etc/sysctl.conf /etc/sysctl.d/*.conf -> r:1'
```

Comment:
This is not a bad solution, but take into account that we are being more permissive now.

Before the change, we force the user to set that flag to 0 to validate the check. Now, it is enough to have any other value than 1.

Reply:
If there is any other value than 1, is like the value was 0. and we will relay on sysctl. seems like the file check will just work after restarting system and we will relay again on sysctl.